x-pack/plugins/monitoring/server/lib/cluster/get_clusters_stats.js

```diff
@@ -83,9 +83,9 @@ export function handleClusterStats(response) {
 
   return hits.map(hit => {
     const cluster = get(hit, '_source');
+    const indexName = get(hit, '_index', '');
 
     if (cluster) {
-      const indexName = get(hit, '_index', '');
```

Comment:
If you add `cluster.indexName = indexName` in this block, then you can avoid changing the return line and the tests will probably be cleaner to update.